automation/patch_igz/patch_remote.py

```diff
@@ -268,113 +280,80 @@ def _push_docker_images(self, built_images):
             )
 
     def _patch_deployment_from_file(self):
-        logger.info("Patching mlrun-api-chief deployment")
-        self._exec_remote(
-            [
-                "kubectl",
-                "-n",
-                "default-tenant",
-                "patch",
-                "deployment",
-                "mlrun-api-chief",
-                "-p",
-                f"{self._deploy_patch}",
-            ]
-        )
-
-        logger.info("Patching mlrun-api-worker deployment")
-        self._exec_remote(
-            [
-                "kubectl",
-                "-n",
-                "default-tenant",
-                "patch",
-                "deployment",
-                "mlrun-api-worker",
-                "-p",
-                f"{self._deploy_patch}",
-            ]
-        )
+        for deployment in self._deployments:
+            logger.info(f"Patching deployment {deployment}")
```

Comment:
```suggestion
            logger.info(f"Patching deployment", deployment=deployment)
```

Reply:
this script has no structured logger but python basic one